app/helpers/promos_helper.rb

```diff
@@ -58,4 +58,17 @@ def facebook_url(referral_code)
   def base_human_referral_url
     Rails.application.secrets[:base_referral_url].to_s + "/"
   end
+
+  def on_channel_type(channel)
```

Comment:
There might be a better name for this method.  Eventually this should move the PublishersHelper, as this should also be displayed on the dashboard next to each channel name.